message ServerMetaData {
    message Sensor {
        required uint32 sensorId = 1;
        optional string sensorName = 2;
    }

    repeated Sensor sensors = 1;
    optional int32 timeZoneOffset = 2;
    optional double longitude = 3;
    optional double latitude = 4;
}

